localization/pose_estimator/ndt_scan_matcher/ndt_omp/apps/align.cpp

```diff
@@ -45,17 +44,16 @@ pcl::PointCloud<pcl::PointXYZ>::Ptr align(
   registration->setInputTarget(target_cloud);
   registration->setInputSource(source_cloud);
   pcl::PointCloud<pcl::PointXYZ>::Ptr aligned(new pcl::PointCloud<pcl::PointXYZ>());
-
-  auto t1 = ros::WallTime::now();
+  auto t1 = std::chrono::steady_clock::now();
   registration->align(*aligned);
-  auto t2 = ros::WallTime::now();
-  std::cout << "single : " << (t2 - t1).toSec() * 1000 << "[msec]" << std::endl;
+  auto t2 = std::chrono::steady_clock::now();
+  std::cout << "single : " << std::chrono::duration_cast<std::chrono::milliseconds>(t2 - t1).count() << "[msec]" << std::endl;
 
   for (int i = 0; i < 10; i++) {
     registration->align(*aligned);
   }
```

Comment:
Maybe outside the scope of porting - but this is strange. Call align once, then call ten more times? Is this setting up the parallelism by calling 10 times, or is it simple meant to iteratively provide a better registration?

Reply:
I think it might be some sort of benchmarking.

Reply:
@nnmm I agree - and don't think this is used very much. As you say below outside the scope of this porting.